pkg/controller/direct/common/labels.go

```diff
@@ -15,32 +15,27 @@
 package common
 
 import (
-	"strings"
+	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/controller/direct"
+	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/label"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
-func ComputeGCPLabels(labels map[string]string) map[string]string {
-	if labels == nil {
-		return map[string]string{}
+// This function should be called if the typed object has `spec.labels` field.
+func ComputeLabels_ToProto(mapCtx *direct.MapContext, u *unstructured.Unstructured) map[string]string {
```

Comment:
Nit: Labels_ToProto (otherwise I think this is specific to the Compute APIs)

Reply:
oh, good point!